sdk_container/src/third_party/coreos-overlay/coreos-devel/sdk-depends/sdk-depends-0.0.1.ebuild

```diff
@@ -43,6 +43,7 @@ DEPEND="
 	sys-apps/seismograph
 	sys-boot/grub
 	amd64? ( sys-boot/shim )
+	amd64? ( sys-boot/shim-signed )
```

Comment:
Shouldn't this be instead of sys-boot/shim?

Reply:
Both the packages actually would be moved to the board packages.